src/widgets/dialogs/UserInfoPopup.hpp

```diff
@@ -35,15 +37,16 @@ class UserInfoPopup final : public BaseWindow
     QString userName_;
     QString userId_;
     ChannelPtr channel_;
+    ChannelView *latestMessages_;

```

Comment:
`ChannelView *latestMessages_{}`

Adding the `{}` sets the default value to `nullptr` instead of a random int which makes debugging easier. (This works for most types from "C" like ints and pointers, C++ types do not need this)